WTWSMS/decisions/realm_decisions.txt

```diff
@@ -662,7 +662,14 @@ decisions = {
 			has_global_flag = viking_age_started
 			NOT = { has_global_flag = varangian_guard_founded }
 			any_playable_ruler = {
-				religion = norse_pagan
+				OR = {
+					religion = norse_pagan
+					religion = germanic_pagan
+					religion = norse_pagan_reformed
+					religion = germanic_pagan_reformed
+					religion = saxon_pagan
+					religion = saxon_pagan_reformed
+				}
```

Comment:
Checking for additional things which we don't use and may be subject for removal is usually a bad idea for maintenance.